typegraph/node/sdk/src/typegraph.ts

```diff
@@ -201,34 +204,34 @@ export function typegraph(
     ref: (name: string) => {
       return genRef(name);
     },
-    configureRandomInjection: (params: {seed: number}) => {
-      return core.setSeed(params.seed);
-    },
+    configureRandomInjection: (params: { seed: number }) => {   
+    return core.setSeed(params.seed);
+  },
     ...InjectionSource,
   };
 
-  builder(g);
-
-  let serialize = () => "";
-  if (!disableAutoSerialization) {
-    const tgJson = core.finalizeTypegraph({ tag: "simple" });
-    console.log(tgJson);
-    serialize = () => tgJson;
-  } else {
-    // config is known at deploy time
-    serialize = (config?: ArtifactResolutionConfig) => {
-      const tgJson = core.finalizeTypegraph({
-        tag: "resolve-artifacts",
-        val: config,
-      });
-      return tgJson;
-    };
-  }
-  return {
-    serialize,
-    name,
+builder(g);
+
+let serialize = () => "";
+if (!disableAutoSerialization) {
+  const tgJson = core.finalizeTypegraph({ tag: "simple" });
+  console.log(tgJson);
+  serialize = () => tgJson;
+} else {
+  // config is known at deploy time
+  serialize = (config?: ArtifactResolutionConfig) => {
+    const tgJson = core.finalizeTypegraph({
+      tag: "resolve-artifacts",
+      val: config,
+    });
+    return tgJson;
   };
 }
+return {
+  serialize,
+  name,
+};
```

Comment:
broken indentation?

Reply:
Oof. I've added the node sdk source files to the pre-commit format hook.